.circleci/config.yml

```diff
@@ -1,54 +0,0 @@
-version: 2.1
```

Comment:
Why do you delete all of those files? I would prefer to have this in a separate PR, named accordingly...

Reply:
We are looking into this, since it was not intended to do it.